crates/pop-contracts/src/utils/contracts_node.rs

```diff
@@ -39,36 +34,66 @@ pub async fn is_chain_alive(url: url::Url) -> Result<bool, Error> {
 	}
 }
 
+/// Checks if the `substrate-contracts-node` binary exists
+/// or if the binary exists in pop's cache.
+/// returns:
+/// - Some("", <version-output>) if the standalone binary exists
+/// - Some(binary_cache_location, "") if the binary exists in pop's cache
+/// - None if the binary does not exist
+pub fn does_contracts_node_exist(cache: PathBuf) -> Option<(PathBuf, String)> {
```

Comment:
Just a refactor comment here: using a struct might make this clearer than using a tuple (path and version), which would likely eliminate the need for all the comments explaining the return values.